site/source/user-manual/flashing/flashing-pi.rst

```diff
@@ -3,7 +3,7 @@
 =======================
 Flashing (Raspberry Pi)
 =======================
-This guide is for flashing StartOS to a micro SD card in order to install it on a Raspberry Pi with an external SSD.  Please contact support if you need to re-install your OS on a Lite (SD card storage).
+This guide is for flashing StartOS to a micro SD card in order to install it on a Raspberry Pi with an external SSD.  Please `contact support <https://start9.com/contact>`_ if you need to re-install your OS on a Lite (SD card storage).
```

Comment:
```suggestion
This guide is for flashing StartOS to a micro SD card in order to install it on a Raspberry Pi with an external SSD.  Please `contact support <https://start9.com/contact>`_ if you need to re-install your OS on a Server Lite (SD card storage).
```

Reply:
Suggestion: express full product name 'Server Lite' to avoid any possible confusion